lib/ord.js

```diff
@@ -40,32 +41,38 @@ const initializeAppConfig = (csn) => {
     for (const key of modelKeys) {
         const keyDefinition = csn.definitions[key];
         switch (keyDefinition.kind) {
-            case ORD_RESOURCE_TYPE.service:
+            case CDS_ELEMENT_KIND.service:
                 if (!keyDefinition["@cds.external"]) {
                     serviceNames.push(key);
                 }
                 break;
             // TODO: should be rewritten
-            case ORD_RESOURCE_TYPE.entity:
-                if (!key.includes(".texts") && keyDefinition["@ODM.entityName"]) {
-                    odmEntity.push(createEntityTypeTemplate(keyDefinition));
+            case CDS_ELEMENT_KIND.entity:
+                if (!key.includes(".texts")) {
+                    endpoints.add(key);
+                    if (keyDefinition["@ODM.entityName"]) {
```

Comment:
I would suggest that uses constant value ORD_EXTENSIONS_PREFIX defined in constans.js

Reply:
This constant/enum is CAP CDS/CSN specific, not directly related to ORD.
Here you can find an explanation why I've decided to give name CDS_ELEMENT_KIND: https://pages.github.tools.sap/cap/docs/cds/csn#definitions. 
There stands: definition `kind` can be: one of context, service, entity, type, action, function, or annotation
Maybe to rename into CSN_DEFINITION_KID or CDS_DEFINITION_KIND? Or you still prefer to rename `CDS_ELEMENT_KIND` into `ORD_EXTENSIONS_PREFIX`?

Reply:
I mean write @ODM.entityName into constant value like ORD_EXTENSIONS_PREFIX.